docs/editors/setup.md

```diff
@@ -372,3 +372,125 @@ Alternatively, it can be used via the [Apheleia](https://github.com/radian-softw
 
 Ruff is also available via the [`textmate2-ruff-linter`](https://github.com/vigo/textmate2-ruff-linter)
 bundle for TextMate.
+
+## Zed
+
+Ruff is available as an extension for the Zed editor. To install it:
+
+1. Open the command palette with `Cmd+Shift+P`
+1. Search for "zed: extensions"
+1. Search for "ruff" in the extensions list and click "Install"
+
+To configure Zed to use the Ruff language server for Python files, add the following
+to your `settings.json` file:
+
+```json
+{
+  "languages": {
+    "Python": {
+      "language_servers": ["ruff"]
+      // Or, if there are other language servers you want to use with Python
+      // "language_servers": ["pyright", "ruff"]
+    }
+  }
+}
+```
+
+To configure the language server, you can provide the [server settings](settings.md)
+under the [`lsp.ruff.initialization_options.settings`](https://zed.dev/docs/configuring-zed#lsp) key:
+
+```json
+{
+  "lsp": {
+    "ruff": {
+      "initialization_options": {
+        "settings": {
+          // Ruff server settings goes here
+          "lineLength": 80,
+          "lint": {
+            "extendSelect": ["I"],
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+!!! note
+    Configuring multiple formatters for a given language requires the [preview
+    version](https://zed.dev/releases/preview) of Zed.
```

Comment:
Maybe we should be specific about the version (0.146.0) multi-formatters were made available in instead? As is, it sounds like I have to use Preview to use multiple formatters, when in reality this change is gonna be available on Stable channel in a week.

Reply:
Yeah, that makes sense, thanks!